exareme2/algorithms/specifications.py

```diff
@@ -102,6 +102,7 @@ class InputDataSpecification(ImmutableBaseModel):
 class InputDataSpecifications(ImmutableBaseModel):
     y: InputDataSpecification
     x: Optional[InputDataSpecification]
+    validation: Optional[bool]
```

Comment:
I think I would prefer for the whole `validation_datasets` specification to be there, like
```
validation_datasets: Optional[InputDataSpecification]
```

WDYT?

Reply:
As we discuss one a call today I will stay as is.